### DISCUSSION TITLE
Bem-Vindo(a) a Comunidade GuiaDoProgramador!

### DISCUSSION THREAD

**Discussion by vncsmnl** [vncsmnl/guiadoprogramador#1] [Announcements]
## 👋 Bem-vindo meus queridos(as)!

Estaremos usando este repositório como um lugar para nos conectarmos com outros devs que estão com sede de conhecimento assim como você!! Espero que você descubra ótimas ferramentas e faça muitas amizades! 💞

Esperamos que você:
  * Faça perguntas sobre as quais você tem duvidas.
  * Compartilhe ideias.
  * Compartilhe o repositório.
  * Se envolva com outros membros da comunidade.
  * Dê boas-vindas aos outros e tenha a mente aberta. Lembre-se que esta é uma comunidade que
  construir juntos 💪.

  Para começar, que tal você se apresentar e nos contar mais sobre por que você se tornou ou gostaria de se tornar um dev. 🌚

**Comment by ghost** [comment]
Oi gente!
